arch/x86/include/intel64/kernel_arch_data.h

```diff
@@ -38,4 +39,43 @@ extern uint8_t x86_cpu_loapics[];	/* CPU logical ID -> local APIC ID */
 #define Z_X86_TRAMPOLINE_STACK_SIZE	128
 #endif
 
+#ifdef CONFIG_X86_KPTI
+#define TRAMPOLINE_STACK(n)									\
+	uint8_t z_x86_trampoline_stack##n[Z_X86_TRAMPOLINE_STACK_SIZE]				\
+		__attribute__ ((section(".trampolines")));
+
+#define TRAMPOLINE_INIT(n)									\
+	.ist2 = (uint64_t)z_x86_trampoline_stack##n + Z_X86_TRAMPOLINE_STACK_SIZE,
+#else
+#define TRAMPOLINE_STACK(n)
+#define TRAMPOLINE_INIT(n)
+#endif /* CONFIG_X86_KPTI */
+
+#define ACPI_CPU_INIT(n, _)									\
+	uint8_t z_x86_exception_stack##n[CONFIG_X86_EXCEPTION_STACK_SIZE] __aligned(16);	\
+	uint8_t z_x86_nmi_stack##n[CONFIG_X86_EXCEPTION_STACK_SIZE] __aligned(16);		\
+	TRAMPOLINE_STACK(n);									\
+	Z_GENERIC_SECTION(.tss)									\
+	struct x86_tss64 tss##n = {								\
+		TRAMPOLINE_INIT(n)								\
+		.ist6 =	(uint64_t)z_x86_nmi_stack##n + CONFIG_X86_EXCEPTION_STACK_SIZE,		\
+		.ist7 = (uint64_t)z_x86_exception_stack##n + CONFIG_X86_EXCEPTION_STACK_SIZE,	\
+		.iomapb = 0xFFFF, .cpu = &(_kernel.cpus[n])}
```

Comment:
have a new line for each attribute. Macro generated code should follow the same code style as the hand written one. (so have an empty line for the end } and properly indented as well.)